.vscode/launch.json

```diff
@@ -5,15 +5,20 @@
     "version": "0.2.0",
     "configurations": [
         {
+            "name": "Debug current test file in VSCode",
```

Comment:
This is global to solution change. Does this work across all packages?

Reply:
It works when you debug unit test in vscode by clicking f5 for any unit test file.